bundle/language_fr.properties

```
@@ -1507,6 +1510,16 @@ angal.pricesothers.othm
 angal.pricesothers.pleaseselectanitemtodelete                                                          = Veuillez sélectionner un élément à supprimer
 angal.pricesothers.pleaseselectanitemtoedit                                                            = Sélectionner un élément à modifier
 angal.pricesothers.thedatacouldnotbedeleted                                                            = La date ne peut pas être supprimée
+angal.reduction.exam                                                                                   = Examen
+angal.reduction.examrate.col                                                                           = Taux examen.
+angal.reduction.medical                                                                                = M\u00E9dical
```

```suggestion
angal.reduction.medical                                                                                = Médical
```
Do the same thing on the other lines.